lib/barcelona/network/vpc_builder.rb

```diff
@@ -375,6 +393,28 @@ def build_resources
         add_resource("AWS::SNS::Topic", "NotificationTopic") do |j|
           j.DisplayName "district-#{stack.district.name}-notification"
         end
+
+        add_resource("AWS::S3::BucketPolicy", "BucketPolicy") do |j|
+          j.Bucket stack.district.s3_bucket_name
+          j.PolicyDocument(
+            {
+              "Statement" => [
+                {
+                  "Action" => ["s3:PutObject"],
+                  "Effect" => "Allow",
+                  "Resource" => join("",
+                                     "arn:aws:s3:::",
+                                     "#{stack.district.s3_bucket_name}/elb_logs/*/AWSLogs/",
```

Comment:
This is a little different from the official manual. Since an S3 bucket can have only one policy, we need to define a bucket policy that enables all ELBs in a district to log.